proto/process/agent.proto

```diff
@@ -720,6 +720,20 @@ message Connection {
 	uint32 stateIndex = 46;
 }
 
+message FailedConnection {
+	int32 pid = 1;
+	Addr laddr = 2; // Local address
+	Addr raddr = 3; // Remote address
+
+	ConnectionFamily family = 4;
+	ConnectionType type = 5;
+	ConnectionDirection direction = 6;
+	uint32 netNS = 7; // Network namespace
```

Comment:
@Yumasi 
would be nice to have `ipTranslation` and `pidCreateTime`
We probably need : `	EphemeralPortState` and `isLocalPortEphemeral = 41;` as well

Any reason to use a new message type FailedConnection vs Connection ?
In case of memory footprint, not sure it's a good argument as there will be more Connection than Failed one.

Reply:
let's keep memory footprint low and the FailedConnection type as previously discussed in previous comment

Reply:
`pidCreateTime` is just used by checks/net